conduct.md

```diff
@@ -88,9 +88,11 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Team members should finalize decisions when the team reaches a majority vote.
+- Team members should consider and discuss all possible ideas of each student before moving forward.
+- Team members should not take final decisions personally and criticize ideas, not team members.
```

Comment:
I think this line would be clearer if it was re-worded a bit. Maybe say something like "and should criticize ideas, rather than team members" at the end instead.

Reply:
"criticize" might not be the best word choice here since it has a negative connotation. Maybe the last part of this statement is not necessary since it does not really relate to the first part of the bullet point?

Reply:
Thanks for the review. How about:

> Team members should not take final decisions personally and understand that the decisions were based on ideas rather than team member's personality.

I'm still not sure whether to include the last part. Could you please suggest better words than "team member's personality"?

Reply:
Agreeing with @mariakimheinert 's comment; I think this line would be fine if you chose to remove the last bit of the sentence:
```suggestion
- Team members should not take final decisions personally.
```